client/features/addToCartButton/AddToCartButton.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectMe } from '../auth/authSlice';
+
+// Bryan, from your product's components you should be sending a product object and a quantity number (can be 1 for now)
```

Comment:
So you want me to pass product and quantity as props into the 'addToCartButton' component when I use it? Might need some clarity on this if you want to talk tomorrow.

Reply:
That's right! Happy to talk this am...